adafruit_circuitplayground/express.py

```diff
@@ -118,6 +118,32 @@ def __init__(self):
         self._lis3dh = adafruit_lis3dh.LIS3DH_I2C(self._i2c, address=0x19)
         self._lis3dh.range = adafruit_lis3dh.RANGE_8_G
 
+        # Initialise tap:
+        self._lis3dh.set_tap(2, 18, time_limit=4, time_latency=17, time_window=110)
+        self._last_tap = False
+
+    @property
+    def double_tap(self):
+        """True once after a double tap.
+
+        .. image :: /_static/accelerometer.jpg
+          :alt: Accelerometer
+
+        Quickly tap the CPX twice to double-tap.
+
+        .. code-block:: python
+
+          from adafruit_circuitplayground.express import cpx
+
+          while True:
+              if cpx.double_tap:
+                  print("Double tap!")
+        """
+        tapped = self._lis3dh.tapped
```

Comment:
Want to catch an attribute error here like shake if the lis3dh library is too old? Maybe make the message generic so its exact between the two.